_pages/en/web-services-access.md

```diff
@@ -1,63 +1,57 @@
 
-### Considerations
+ESDC is currently blocking access to some popular and useful collaboration Web tools and SaaS like Google Docs, Google Hangouts and OneDrive (see [Appendix A](#appendix-a---web-tools-currently-blocked)).
+The rationale for blocking them is primarily to prevent an accidental leak of protected information by employees.
 
-Departments are leveraging Web tools to collaborate and engage with citizens and public administrations both in Canada and across the globe.
-For example, the latest Open Government Plan and Service and Digital Target Architecture from TBS were shared as drafts for consultation and comments on Google Docs.
-The GC Digital Academy and Free Agents, as well as other groups, frequently use Google Forms for registration.
+A proposed mitigation item to reduce this risk is to augment the Data Loss Prevention (DLP) capability with a tool that enforces classification of documents and emails.
+However, implementation is not being considered for the short term.
 
-In order to improve ESDC's ability to leverage current Web tools, a balanced approach is proposed for granting access to additional Web tools (See [Appendix A](#appendix-a---web-tools-to-grant-access)) only for IITB employees in the short term, as a step toward broader access in ESDC.
-As IITB employees generally don't deal with protected information and are well informed and trained on the management of information, the risk would be minimal.
-In addition, the benefits should bring great value such as enhancing collaboration and efficiency, increasing job satisfaction and helping attract and retain IT talent.
+A firewall upgrade is underway and configuration for categories and sites blocked or unblocked will be documented and reviewed.
 
 ### Recommendations
 
 To support the adoption of the new [Policy on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32603), ESDC needs to:
 
-- Grant access to additional Web tools (see [Appendix A](#appendix-a---web-tools-to-grant-access)) for IITB employees, as a step toward broader open access to the Internet for ESDC employees;
-- Perform an evaluation of security monitoring tools, active at the source (application and database) and the perimeter (DLP), to mitigate future risks associated with broader open access to the Internet; and
+- Grant access to the following Web site categories (or related categories) for IITB employees: Instant Messaging, Document Editing, Web Conferencing, Email and File Storage;
+- Ensure IITB employees' understanding of document classification and that Web tools available on the Internet are only to be used with unclassified data and information;
+- Perform an evaluation of security monitoring and classification tools to mitigate future risks associated with broader open access to the Internet; and
 - Review access rights to data and information.
 
-### Next Steps
-
-The IT Strategy team will work with teams in IITB to:
+## Appendix A - Web Tools Currently Blocked
```

Comment:
The Next steps are vague. Need to be clear on what we will be asking  Director/DG Comms on what they will commit to.
- Which teams are involved in the decision? How will we [engage them and get their endorsement](https://sara-sabr.github.io/ITStrategy/strategy-communications.html)?
- How will we ensure the organization still complies with the [Directive on Security Management](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32611), especially its Appendix E on Mandatory Procedures for IM Security Controls (referenced in the Directive on Service and Digital)
- We will review the following ESDC and IITB policy instruments
-- [Network Use Directive](http://iservice.prv/eng/is/security/tools_and_resources/toolkit/network_use_directive.shtml)
-- [Storage of Electronic Information Directive](http://iservice.prv/eng/is/security/tools_and_resources/storage_electronic_Info_directive.shtml)

Reply:
Is it all IITB employees that we target? Or a sub-group of them? How do we profile them (and how does the system/Active Directory GPOs/firewall be able to manage their profile identities)? Are we asking someone in Operations to manually tag IITB employees or will this be something we will take from the HR system?

More effort needs to take place to identify how this will work. At this stage, this may mean engaging with operations and IT security teams to determine how.

This will mean asking for their time and efforts. Which they don't have right now, unlesss we prioritize this strategy above other priorities. To do this, we need to have very strong arguments.

Reply:
There are no Next steps in this version.  Following BN format, Next steps are for information, these are recomendations